cmd/lakectl/cmd/dbt.go

```diff
@@ -1,36 +1,91 @@
 package cmd
 
 import (
-	"bufio"
-	"bytes"
 	"context"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path"
 	"regexp"
-	"strings"
 
 	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/dbtutil"
 	"github.com/treeverse/lakefs/pkg/metastore"
 	mserrors "github.com/treeverse/lakefs/pkg/metastore/errors"
 )
 
+type DbtCommandClient interface {
```

Comment:
```suggestion
type DBTCommandClient interface {
```
(and in other `type`s and `func`s below): the Go way is to keep capitalization of acronyms and abbreviations.  So we always have `XyzzyID` (with a capital D), etc.